external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -315,14 +315,15 @@ public void createFullTrigger(RoutingContext routing) {
             @DocResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ResponseUtil.ApiError.class),
             @DocResponse(code = 500, message = "Internal server error.", response = ResponseUtil.ApiError.class)
     })
-    @Timed(absolute = true, tags = {"path=/triggers/trigger/{triggerId}", "method=PUT"})
+    @Timed
+            //(absolute = true, tags = {"path=/triggers/trigger/{triggerId}", "method=PUT"})
```

Comment:
Are the old tags automatically added by Micrometer? Should that comment be removed?

Reply:
I tried to somehow add the tags with the last cmmit, but absolute = true isn't available